Ombi.UI/Modules/SearchModule.cs

```diff
@@ -222,6 +229,36 @@ public class SearchModule : BaseAuthModule
             return await ProcessMovies(MovieSearchType.Search, searchTerm);
         }
 
+        private async Task<Response> SearchActor(string searchTerm)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<Response> SearchActor(string searchTerm, bool filterExisting)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm, AlreadyAvailable).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<bool> AlreadyAvailable(int id, string title, string year)
+        {
+            await Task.Yield();
+            return IsMovieInCache(id, String.Empty) || PlexChecker.IsMovieAvailable(plexMovies(), title, year);
+        }
+
+        private IEnumerable<PlexContent> plexMovies()
```

Comment:
I think we should stick to the way the search currently works. It's a good idea, but I think it might bring on some confusion.